pom.xml

```diff
@@ -19,27 +20,27 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.66</version>
+        <version>4.75</version>
         <relativePath />
     </parent>
 
     <artifactId>fortify</artifactId>
-    <version>23.1.40-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
 
     <properties>
-        <gitHubRepo>jenkinsci/fortify-plugin</gitHubRepo>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
The latest plugin parent POM already sets this, so no need to duplicate it here.